jsonnet/rules.jsonnet

```diff
@@ -3,6 +3,26 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
 {
   prometheusRules+:: {
     groups+: [
+      {
+        name: 'openshift-general.rules',
+        rules: [
+          {
+            expr: |||
+              100 * (count(up == 0 unless on (node) max by (node) (kube_node_spec_unschedulable == 1)) BY (job, namespace, service) /
+                count(up unless on (node) max by (node) (kube_node_spec_unschedulable == 1)) BY (job, namespace, service)) > 10
+            |||,
+            alert: 'TargetDown',
+            'for': '15m',
+            annotations: {
+              description: '{{ printf "%.4g" $value }}% of the {{ $labels.job }}/{{ $labels.service }} targets in {{ $labels.namespace }} namespace have been unreachable for more than 60 minutes. This may be a symptom of network connectivity issues, down nodes, or failures within these components. Assess the health of the infrastructure and nodes running these targets and then contact support.',
```

Comment:
Time doesn't match the "for" in the rule

Reply:
```suggestion
              description: '{{ printf "%.4g" $value }}% of the {{ $labels.job }}/{{ $labels.service }} targets in {{ $labels.namespace }} namespace have been unreachable for more than 15 minutes. This may be a symptom of network connectivity issues, down nodes, or failures within these components. Assess the health of the infrastructure and nodes running these targets and then contact support.',
```